<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkspaceDialog</class>
 <widget class="QDialog" name="WorkspaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select PiSiDo Workspace</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Please select a workspace. All packages and their build files will be created in here.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lbl_workspace">
       <property name="text">
        <string>Workspace</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_workspace">
       <property name="toolTip">
        <string>Produced build files ad pisi packages will save under here</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_browse">
       <property name="text">
        <string>&amp;Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>33</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QCheckBox" name="check_not_ask">
     <property name="toolTip">
      <string>You can change workspace form menu</string>
     </property>
     <property name="text">
      <string>Don't ask workspace again.</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>251</width>
         <height>19</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_ok">
       <property name="text">
        <string>&amp;Ok</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_exit">
       <property name="toolTip">
        <string>Exit application</string>
       </property>
       <property name="text">
        <string>E&amp;xit</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>le_workspace</tabstop>
  <tabstop>pb_browse</tabstop>
  <tabstop>check_not_ask</tabstop>
  <tabstop>pb_ok</tabstop>
  <tabstop>pb_exit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pb_ok</sender>
   <signal>clicked()</signal>
   <receiver>WorkspaceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_exit</sender>
   <signal>clicked()</signal>
   <receiver>WorkspaceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
